tests/unit/test_credentials.py

```diff
@@ -257,6 +265,7 @@ def get_expected_creds_from_response(self, response):
             'secret_key': response['Credentials']['SecretAccessKey'],
             'token': response['Credentials']['SessionToken'],
             'expiry_time': expiration,
+            'account_id': None,
```

Comment:
This change is required since account ID is not conditionally added to the creds dict. We haven't needed to conditionally set params to it in the past and I chose not to here. If it's preferred it's a simple change.

Reply:
This seems like a risk. Why do the responses have to include this value now? Can anything subclassing credentials or response structure that may not return this format break?

Reply:
Yeah I see what you're getting at here. I'll shoot up an update that only sets `account_id` if the value has been successfully resolved to a value that isn't `None`.